repository/scripts/legacy/interaction/Stairs.js

```diff
@@ -1,169 +0,0 @@
-/**
```

Comment:
there was only one door added and I thought if the system for doors has to be rewritten there was no point in having one door. the stairs I was going to re add them as I add in areas